source/atomics.tex

```diff
@@ -154,7 +154,7 @@
   @\placeholdernc{integral}@ atomic_fetch_xor_explicit(volatile @\placeholder{atomic-integral}@*@\itcorr[-1]@, @\placeholdernc{integral}@, memory_order) noexcept;
   @\placeholdernc{integral}@ atomic_fetch_xor_explicit(@\placeholder{atomic-integral}@*@\itcorr[-1]@, @\placeholdernc{integral}@, memory_order) noexcept;
 
-  // \ref{atomics.types.operations.pointer}, partial specializations for pointers
+  // \ref{atomics.types.operations.pointer}, nonmember overloads for atomic<T*>
   template <class T>
```

Comment:
29.6.4 [atomics.types.operations.pointer] is the section referred to, and it also talks about partial specializations. These should be fixed together.